student.rb

```diff
@@ -1,6 +1,8 @@
 require 'person'
```

Comment:
- [ ] Kindly use `require_relative` instead of `require` in student.rb and teacher.rb files to avoid this kind of error 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/218267007-526e06ff-840e-484d-875b-81d7074beb6d.png)